djangocms_versioning_filer/cms_config.py

```diff
@@ -23,11 +27,25 @@
     FilerContentConfig = None
 
 
+def delete_published_thumbnail(file_obj):
```

Comment:
can you please elaborate why this is not using the already existing function to delete thumbnails? https://github.com/django-cms/django-filer/blob/master/filer/models/filemodels.py#L232-L237

Reply:
You are right @kinkerl , I should use the existing method. I recreate a image locally and publish, now I can find the related thumbnail image via `get_source_cache` and `source_cache.thumbnails.all()`. 
I will update this.